department-of-veterans-affairs/caseflow
# [PART 2] Allow attorney and judge on hold and completed tabs to be paginated

Bumps #14142

PART 2 of the stack
Part 1: #14428
PART 3: #14468
PART 4: #14486

### Description
Does not allow the pagination of attorney and judge assigned tabs on the back end

### Acceptance Criteria
- [ ] Even with pagination on, attorney and judge queue return full set of tasks to the front end

### Testing Plan
1. Enable pagination
   ```ruby
   FeatureToggle.enable!(:user_queue_pagination)
   ```
1. Sign is as Steve Casper
1. Go to your queue
1. Ensure queue config returns ALL tasks for the user assigned tab
1. Ensure queue config returns paginated tasks for the user on hold tab and completed tab
![Screen Shot 2020-06-09 at 3 07 59 PM](https://user-images.githubusercontent.com/45575454/84189574-759e1500-aa63-11ea-9516-69cae0a03e9c.png)


### User Facing Changes
